ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -295,3 +296,83 @@ def getFilterConfig(filterID):
   :params filterID: string representing a filter identifier.
   """
   return gConfig.getOptionsDict('Resources/LogFilters/%s' % filterID)
+
+
+def getProxyProviders():
+  """ Return list of all proxy provider names defined in the CS
+
+      :return: S_OK(list)/S_ERROR
+  """
+  return gConfig.getSections('%s/ProxyProviders' % gBaseResourcesSection)
+
+
+def getProxyProviderOption(proxyProvider, option):
+  """ Get the option of the proxy provider
+
+      :param basestring proxyProvider: proxy provider name
+      :param basestring option: option that need to get
+
+      :return: basestring -- option value
+  """
+  return gConfig.getValue("%s/ProxyProviders/%s/%s" % (gBaseResourcesSection, proxyProvider, option))
+
+
+def getProxyProviderDict(proxyProvider):
```

Comment:
unused

Reply:
This logic moved to new method getInfoAboutProviders